src/js/components/FormField/__tests__/FormField-test.tsx

```diff
@@ -632,4 +632,27 @@ describe('FormField', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('Custom theme TextInput FormField container', () => {
+    const { container } = render(
+      <Grommet
+        theme={{
+          formField: {
+            textInput: {
+              container: {
+                extend: (props: any) => css`
+                  color: ${props['href'] ? 'green' : 'red'};
```

Comment:
are we just showing here how we can use props? I just saw this and thought why would a textInput have `href` or am I thinking of this wrong

Reply:
Oops this should be disabled not href, I'll go back and fix

Reply:
all good!